UnityProject/Assets/Scripts/Managers/InGameTimeManager.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using AdminCommands;
+using Shared.Managers;
+using UnityEngine;
+using Random = UnityEngine.Random;
+
+namespace Managers
+{
+	public class InGameTimeManager : SingletonManager<InGameTimeManager>
+	{
+		public DateTime UniversalSpaceTime { get; set;}
+		public DateTime UtcTime { get; private set; }
+		public Action OnUpdateTime;
+		private const int GAME_YEAR = 2562;
+
+
+		public override void Awake()
+		{
+			base.Awake();
+			EventManager.AddHandler(Event.RoundStarted, ServerSetupUniversalSpaceTime);
+			UtcTime = DateTime.UtcNow.AddYears(GAME_YEAR);
```

Comment:
wouldn't this mean ``2022 + 2562`` ?

Reply:
uhhhhm I forgor. Maybe it would be nice to have it read from server configuration in the future, so servers can set the initial time as they wish.